scripts/contentScript.js

```diff
@@ -497,6 +500,8 @@ const bitBucketHunkHighlight = (apiResponses) => {
 									const lineNumber = ariaLabel.substring(8);// because aria label = "To line 3273", so removing first 8 letters to get line number 
 									if (parseInt(lineNumber) >= parseInt(foundFiles.line_start) && parseInt(lineNumber) <= parseInt(foundFiles.line_end)) {
 										toLineElement.style.backgroundColor = '#c9cbff';
+										let firstChild = item.firstElementChild;
+										firstChild.style.backgroundColor = '#c9cbff'; // highlight entire line
```

Comment:
No line number highlighting here? Also can you attach a small screenshot to the PR description so I can see the colours